pds4indextools/pds4_create_xml_index.py

```diff
@@ -1373,7 +1376,13 @@ def main(cmd_line=None):
 
         # In this case, this filename is the filename of the index file previously
         # generated.
-        filename = str(Path(index_file).stem)
+        try:
+            filename = str(Path(index_file).stem)
+        except TypeError:
+            print('Label not generated. The "--output-index-file" argument is '
```

Comment:
Do we want to create a label file if we get the default `index<n>.csv` index file? It seems reasonable that we would get `index<n>.xml`.

Reply:
It already does that:

```
(.venv) emiliesimpson@Emilies-MBP rms-pds4indextools % python pds4indextools/pds4_create_xml_index.py test_files/labels 'tester_label_1.xml' --generate-label Product_Metadata_Supplemental
Index file generated at index1.csv
Product_Metadata_Supplemental label generated at index1.xml
```

Reply:
Then the error message you're printing is incorrect. It says `--output-index-file` is required, but it isn't.

Reply:
I see what you're saying now. Hmmm.

On one hand, I don't really see the harm in allowing label generation for default index files. On the other hand, the resulting label files will have very generic identifying information. Their logical identifier could easily be:

`urn:nasa:pds:rms_metadata:document_opus:index1`

Which is not informative at all.

Reply:
How about a warning message instead that says something like "Warning: Because --output-index-file was not specified, the generated label will only contain generic information. When creating index and label files for production use, please specify --output-index-file."

Reply:
Should this be put alongside the "generated label at place" message, or right at the beginning with the `if args.generate_label` check?

Reply:
Probably next to the generated label at place message. Not that they're going to be very far apart in the output.